### PR TITLE
Add extra pages to cli

### PR DESCRIPTION
**Pull request by nappex**

### CODE REVIEW

File: freezeyt/cli.py [hunk 1]
```diff
@@ -9,7 +9,8 @@
 @click.argument('module_name')
 @click.argument('dest_path')
 @click.option('--prefix', default='http://localhost:8000/', help='URL of the application root')
-def main(module_name, dest_path, prefix):
+@click.option('--extra-pages', multiple=True, help='Pages without any link in application')
```
Comment (encukou) [inline freezeyt/cli.py:12]: This is used as `python -m freezeyt --extra-pages page1 --extra-pages page2 ...`, right?
`extra-page` would be a better name.

Comment (nappex) [reply freezeyt/cli.py]: I've renamed.

**Comment by nappex** [comment]
Changes in code was tested by this command:

```
$ python -m freezeyt tests.fixtures.app_with_extra_page.app extra_page_dir/ --extra-page '/extra/'
```
variable `extra_pages` in app.py was commented at the moment of testing.

Another tests for CLI will be added by extra PR to different issue

**Review by encukou — APPROVED** [review]

**Comment by encukou** [comment]
Thanks!

**Pull request merged**